### PR TITLE
Disable timeout

### PR DESCRIPTION
**Pull request by joeyfigaro**
@pburtchaell Need this now, so tossing up this PR with the changes cp added.

### CODE REVIEW

File: test/notification.js [hunk 1]
```diff
@@ -32,48 +29,6 @@ describe('<Notification />', () => {
     expect(component).to.have.className('notification-bar');
   });
 
-  it('has custom class name', () => {
```
Comment (pburtchaell) [inline test/notification.js]: Why did you remove this test?

**Comment by pburtchaell** [comment]
@joeyfigaro Thanks for the PR. I'm happy to see this get resolved. Can you update your commit with proper formatting? The spacing is off in places. I'm also curious why you removed the "has custom class name" tests.

**Comment by pburtchaell** [comment]
It also looks like one of the tests aren't passing.

**Comment by pburtchaell** [comment]
@joeyfigaro Hey! Any updates on this? Thanks!

**Comment by joeyfigaro** [comment]
@pburtchaell a million apologies for leaving this hanging, man. Got pushed onto some other tasks at work, but circling back around now. Taking care of your comments now!

**Comment by joeyfigaro** [comment]
@pburtchaell Alright, I've attempted to cobble together the changes from @cp and the missing tests from master here. I fiddled with the problematic test for a bit and can't figure out why. I did find a typo that I took care of, but that's it for now. Ideas?

**Comment by joeyfigaro** [comment]
Also, `handleDismiss.calledOnce` *is* true when it gets to its assertion.

**Comment by pburtchaell** [comment]
I'm not sure what's up with the test, @joeyfigaro, but thanks for making the other changes! I'll take a look at the failing test once I have some free time. Is that test passing for you locally?

**Comment by joeyfigaro** [comment]
@pburtchaell Excellent, thanks! It isn't passing, no.

**Comment by pburtchaell** [comment]
And [`master` branch is passing tests](https://travis-ci.org/pburtchaell/react-notification/branches), so your diff is breaking this. It would make sense that this test is failing too, since it is related to timeout. Once this is resolved, I can merge in.

File: test/notification.js [hunk 2]
```diff
@@ -206,18 +205,18 @@ describe('<Notification />', () => {
     }, mockNotification.dismissAfter);
   });
 
-  it('onDismiss fires correctly without prop change', done => {
```
Comment (joeyfigaro) [inline test/notification.js]: Not sure why this was removed. Do we want to put this back?

**Comment by joeyfigaro** [comment]
@pburtchaell Tests are passing. :)

File: test/notification.js (hunk 2 above)
Comment (pburtchaell) [inline test/notification.js]: I think you just changed `'onDismiss fires correctly without prop change'` to `'onDismiss fires once when dismissAfter is passed'`. That's fine.

**Comment by pburtchaell** [comment]
@joeyfigaro Great! I will review in-depth as soon as possible and then I'll merge.

**Comment by pburtchaell** [comment]
This is released as `react-notification@6.2.0`. Thank you!
